crates/testing/src/overall_safety_task.rs

```diff
@@ -366,7 +366,7 @@ impl Default for OverallSafetyPropertiesDescription {
             check_leaf: false,
             check_state: true,
             check_block: true,
-            num_failed_views: 0,
+            num_failed_views: 5,
```

Comment:
Why do we increase this to 5?  I feel like it _should_ be 0, and if tests fail then we have some error in our logic.

Reply:
Good catch! Moved back to 0. The reason is that before polling strict testing parameters from `main` to `develop`, this number is set to 5. And when running this PR I accidentally changed some numbers on this branch to see whether switching to the old one could make the test results better. Agree that it should always be 0.